src/modules/components/proposal/proposalActions/proposalActionsTypes/proposalActionTokenWithdraw.ts

```diff
@@ -0,0 +1,51 @@
+import type { ICompositeAddress } from '../../../../types';
```

Comment:
This interface can be removed as we already have one on the codebase

Reply:
This is the existing from the codebase?